pages/techgenius/tg-installation.md

```diff
@@ -74,7 +74,7 @@ cd ole--vagrant-vi
 vagrant up
 ```
 
-Note: Refer to the Prerequisites section of [Vagrant instructions](vagrant.md) if you encountered an error while executing the command `vagrant up`
+Note: Refer to the Prerequisites section of [Vagrant instructions](http://open-learning-exchange.github.io/#!pages/vi/vi-vagrant.md) if you encountered an error while executing the command `vagrant up`
```

Comment:
It would be better to use to relative link here.

Related to https://github.com/open-learning-exchange/open-learning-exchange.github.io/pull/1323#pullrequestreview-69661276

Reply:
@xyb994  I have changed to relative url. Please review the same.